testing/tests/DevExpress.ui.widgets/dropDownButton.tests.js

```diff
@@ -633,6 +633,16 @@ QUnit.module('common use cases', {
         assert.strictEqual(getActionButton(this.dropDownButton).text(), 'Trial for Visual Studio', 'action button has been changed');
     });
 
+    QUnit.test('deferRendering should not do anything if popup has already been rendered', function(assert) {
```

Comment:
Wouldn't it be better to check the popup element strict equality between deferRendering changes?

Reply:
It's good idea, thanks

Reply:
I meant popup elements :)

Reply:
I understood, and even tested it in the playground, but it didn't stop me from screwing up :-D